data/json/monsters/zed_misc.json

```
@@ -527,6 +527,132 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_zombie_nomex_1",
+    "type": "MONSTER",
+    "name": { "str": "nomex zombie" },
+    "description": "This was at some point a zombie wearing some kind of fire-resistant protective gear.  That same outfit is now shredded and has become part of its outer tissue, making it more resilient against physical attacks and heat-induced damage.  It also seems to be rather muscle packed.  At least its overgrown and hardened faceplate still holds up, preventing it from biting you and also rendering its eyes impaired.  However, it looks like it can still hear and smell you just fine.",
```

```suggestion
    "description": "This was at some point a zombie wearing some kind of fire-resistant protective gear.  That same outfit is now shredded and has become part of its outer tissue, making it more resilient against physical attacks and heat-induced damage.  It also seems to be rather muscle packed.  At least its overgrown and hardened faceplate still holds up, preventing it from biting you and also rendering its eyes impaired.  However, it looks like it can still hear and smell you just fine.",
```

How would you know it can hear & smell you if the faceplate is overgrown 🤔

---

When I designed that, I was inspired by the headless zombie evolution line and imagined it a little something like this: When the faceplate of the zombie gets hardened and overgrown, its ability to see gets impaired. As a result, the blob does as usual, making the best of what it got. Since the helmet with a faceplate is not a perfect isolation barrier, with little gaps and cracks opening up the inside to the outside world, it would just greatly improve the senses of hearing and smell of the zombie to overcome the barrier and/or rout some of those sensors directly to those cracks and gaps to make them more effective.
Sure, it could also make some new outside eyes grow, but since those are fairly complex and expensive organs, it just can't be bothered, y'know?

From a balance perspective, it makes up for the zombies shortcomings in other areas.

If it is too much handwavium, I could change it to be different.